client/src/modules/Score/components/ScoreTable/index.tsx

```diff
@@ -105,6 +104,7 @@ export function ScoreTable(props: Props) {
           isVisible: !notVisibleColumns.includes(String(task.id)),
         }));
       setStudents({ ...students, content: courseScore.content, pagination: courseScore.pagination });
+      setSummaryData(studentCourseScore);
```

Comment:
I'm confused by the naming here. Why is it `summary Data` if it's the score of just one student? It could be renamed, for example, to `setStudentScore`

Reply:
agree, renamed